common/lib/capa/capa/responsetypes.py

```diff
@@ -831,26 +854,146 @@ def assign_choice_names(self):
             if not choice.get('id'):
                 choice.set("id", chr(ord("A") + index))
 
-    def get_score(self, student_answers):
+    def grade_via_every_decision_counts(self, **kwargs):
+        """
+        Calculates partial credit on the Every Decision Counts scheme.
+        For each correctly selected or correctly blank choice, score 1 point.
+        Divide by total number of choices.
+        Arguments: all_choices, student_answer, student_non_answers
```

Comment:
Could you document what `all_choices`, `student_answer`, and `student_non_answers` are supposed to be (types, etc). I wasn't sure what `student_non_answers` was supposed to be referring to until I read the code further down where we generate them.